README.md

```diff
@@ -66,3 +66,7 @@ If you'd like to enable debug logs at runtime, pass an environment variable of `
 ```bash
 docker run -e LOG_LEVEL=DEBUG -v $(pwd):/iac seiso/easy_infra terraform validate
 ```
+
+### Automation with GitHub Actions
+
+A basic GitHub action defaults to passing command `terraform validate` to the container. This command is optional to allow using non-Terraform actions as well.
```

Comment:
We should document how someone would actually use this.  What is the relevant config they should add, and where?  Consider looking at the documentation in [the goat](https://github.com/seisollc/goat) for inspiration.